package.json

```diff
@@ -81,12 +81,14 @@
     "identity-obj-proxy": "^3.0.0",
     "jest": "^24.9.0",
     "mini-css-extract-plugin": "^0.9.0",
+    "mutationobserver-shim": "^0.3.7",
```

Comment:
Not sure why I had to add this; it had something to do with async/await. Also not sure why placing the import in setupTests.js had no effect, while placing it directly in the component did. Might setupTests.js not be getting initialized correctly?

Reply:
Hey @JCarmine that's odd but I did have to add some babel plugins to my PR as well to add async/await functionality.